doc/getting_started/index.rst

```diff
@@ -251,41 +213,23 @@ directory using west:
 Export Zephyr CMake package
 ***************************
 
-Exporting Zephyr as a :ref:`cmake_pkg` makes it possible for CMake to automatically find and load
-boilerplate code for building a Zephyr application.
-
-Zephyr CMake package is exported with the following command
-
-.. tabs::
-
-   .. group-tab:: Ubuntu
-
-      .. code-block:: bash
+The :ref:`Zephyr CMake package <cmake_pkg>` allows CMake to load boilerplate
+code required for building Zephyr applications.
```

Comment:
Nothing wrong here, but even if you don't export, then CMake can still load the boilerplate code when using `ZEPHYR_BASE`.
So the main benefit here is 
```suggestion
The :ref:`Zephyr CMake package <cmake_pkg>` allows CMake to automatically load boilerplate
code required for building Zephyr applications.
```